### PR TITLE
devops(event): refactor to model lib

### PR DESCRIPTION
**Pull request by phpgeekfr**

### CODE REVIEW

File: apps/backend-functions/src/invitation.ts [hunk 1]
```diff
@@ -16,25 +19,21 @@ import {
 } from '@blockframes/model';
 import { getOrInviteUserByMail } from './internals/users';
 import { ErrorResultResponse } from './utils';
-import { CallableContext } from "firebase-functions/lib/providers/https";
+import { CallableContext } from 'firebase-functions/lib/providers/https';
```
Comment (phpgeekfr) [inline apps/backend-functions/src/invitation.ts:22]: I like this ;)

File: apps/backend-functions/src/invitation.ts [hunk 2]
```diff
@@ -141,10 +139,13 @@ export interface UserInvitation {
...
-
-  if (!context?.auth) { throw new Error('Permission denied: missing auth context.'); }
+  if (!context?.auth) {
+    throw new Error('Permission denied: missing auth context.');
+  }
   const user = await getDocument<PublicUser>(`users/${context.auth.uid}`);
-  if (!user.orgId) { throw new Error('Permission denied: missing org id.'); }
+  if (!user.orgId) {
+    throw new Error('Permission denied: missing org id.');
+  }
```
Comment (phpgeekfr) [inline apps/backend-functions/src/invitation.ts:148]: but not this

I prefere :
`if (!context?.auth) throw new Error('Permission denied: missing auth context.');` without brackets

File: apps/backend-functions/src/invitation.ts [hunk 3]
```diff
@@ -180,13 +180,9 @@ export const inviteUsers = async (data: UserInvitation, context: CallableContext
...
     const { type, mode, fromOrg } = invitation;
-    const eventData = type == 'attendEvent' ? getEventEmailData({ event, orgName: orgName(fromOrg, 'full'), email, invitationId }) : undefined;
-    const user = await getOrInviteUserByMail(
-      email,
-      { id: invitationId, type, mode, fromOrg },
-      data.app,
-      eventData
-    )
+    const eventData =
+      type == 'attendEvent' ? getEventEmailData({ event, orgName: orgName(fromOrg, 'full'), email, invitationId }) : undefined;
```
Comment (phpgeekfr) [inline apps/backend-functions/src/invitation.ts:184]:
```suggestion
    const eventData = type == 'attendEvent' ? getEventEmailData({ event, orgName: orgName(fromOrg, 'full'), email, invitationId }) : undefined;
```

File: apps/backend-functions/src/invitation.ts [hunk 4]
```diff
@@ -253,10 +250,11 @@ export const acceptOrDeclineInvitationAsAnonymous = async (data: AnonymousInvita
 
   if (!invitation || invitation.type !== 'attendEvent') throw new Error('Permission denied: invalid invitation');
 
-  if (invitation.mode !== 'invitation' || invitation.toUser.email.toLowerCase() !== data.email.toLowerCase()) throw new Error('Permission denied: invalid invitation');
+  if (invitation.mode !== 'invitation' || invitation.toUser.email.toLowerCase() !== data.email.toLowerCase())
+    throw new Error('Permission denied: invalid invitation');
```
Comment (phpgeekfr) [inline apps/backend-functions/src/invitation.ts:254]:
```suggestion
  if (invitation.mode !== 'invitation' || invitation.toUser.email.toLowerCase() !== data.email.toLowerCase()) throw new Error('Permission denied: invalid invitation');
```

File: apps/festival/festival-e2e/src/fixtures/screening-events.ts [hunk 5]
```diff
@@ -1,5 +1,4 @@
-import type { Screening } from '@blockframes/event/+state/event.firestore';
-import type { Movie, Organization, User} from '@blockframes/model';
+import type { Screening, Movie, Organization, User} from '@blockframes/model';
```
Comment (phpgeekfr) [inline apps/festival/festival-e2e/src/fixtures/screening-events.ts:1]: missing space

**Review by phpgeekfr — COMMENTED** [review]
please be careful with the changes that your formatter do automatically

**Review by KvnMrs — APPROVED** [review]

**Review by KevinDurand974 — APPROVED** [review]

**Pull request merged**